t/test_postgresql

```diff
@@ -14,13 +14,20 @@ fi
 if test -d $DIR; then
   rm -r $DIR
 fi
-initdb --auth-local=peer -N $DIR -U $USER
+initdb --auth-local=peer -N $DIR -U $(whoami)
 echo "listen_addresses=''" >> $DIR/postgresql.conf
 echo "unix_socket_directories='$DIR'" >> $DIR/postgresql.conf
 echo "fsync=off" >> $DIR/postgresql.conf
 echo "full_page_writes=off" >> $DIR/postgresql.conf
 
-pg_ctl -D $DIR -l log_test_postgresql start -w
+LOGDATE=$(date --iso-8601=seconds)
+LOGDIR="$DIR/log"
+LOGFILE="$LOGDIR/postgresql-$LOGDATE.log"
+echo "Logs will be stored in $LOGFILE."
```

Comment:
Should this maybe say `PostgreSQL logs will be stored...`, just to be 100% clear?